ph5/utilities/segd2ph5.py

```diff
@@ -227,6 +243,14 @@ def get_args():
     else:
         raise Exception("No outfile (PH5) given.\n")
 
+    # from_mini and num_mini must not coexist
+    if (NUM_MINI is not None) and (FROM_MINI is not None):
+        raise Exception('Option -M and option -F must not be '
+                        'used at the same time.')
+
+    if ONE_PER_MINI and (FROM_MINI is None):
+        raise Exception('Option -O must be associated with option -F.')
```

Comment:
Mutual exclusion can be handled by argparse
https://docs.python.org/3/library/argparse.html#mutual-exclusion